blueprints/02-at-scale/k8s/openldap-stack-values.yml

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) CloudBees, Inc.
+
+#https://artifacthub.io/packages/helm/helm-openldap/openldap-stack-ha
+
+global:
+  adminUser: "admin"
+  adminPassword: ${password}
+  configUser: "admin"
+  configPassword: ${password}
+
+customLdifFiles:
+  00-root.ldif: |-
+    dn: dc=example,dc=org
+    o: Example, Inc
+    objectclass: dcObject
+    objectclass: organization
+  #CloudBees RBAC groups in CasC are mapped to the following LDAP groups.
+  01-groups.ldif: |-
+    dn: ou=Groups,dc=example,dc=org
+    changetype: add
+    objectclass: organizationalUnit
+    ou: Groups
+
+    dn: cn=CI_Admins,ou=Groups,dc=example,dc=org
+    changetype: add
+    cn: CI_Admins
+    objectclass: groupOfUniqueNames
+    uniqueMember: cn=${admin_user_outputs},dc=example,dc=org
+    uniqueMember: cn=admin_cbci_b,dc=example,dc=org
+
+    dn: cn=Dev_Team_X,ou=Groups,dc=example,dc=org
+    changetype: add
+    cn: Dev_Team_X
+    objectclass: groupOfUniqueNames
+    uniqueMember: cn=developer_1,dc=example,dc=org
+    uniqueMember: cn=developer_2,dc=example,dc=org
+
+    dn: cn=Dev_Team_Y,ou=Groups,dc=example,dc=org
+    changetype: add
+    cn: Dev_Team_Y
+    objectclass: groupOfUniqueNames
+    uniqueMember: cn=developer_3,dc=example,dc=org
+    uniqueMember: cn=developer_4,dc=example,dc=org
+  #Use the cn users and global password to sign in to CloudBees CI. 
+  02-users.ldif: |-
+    dn: cn=developer_1,dc=example,dc=org
+    changetype: add
+    objectclass: inetOrgPerson
+    cn: developer_1
+    givenname: developer_1
+    sn: Developer 1
+    displayname: Developer User 1
+    mail: developer.1@gmail.com
```

Comment:
I'm pretty sure that these will are real email addresses. I will create an issue to offer a replacement here. Either custom domain, or potentially even a `+` address solution such as `myuser+dev1@acme.org`, etc.